docs/modules/c-sharp/index.md

```diff
@@ -154,18 +154,34 @@ SpacetimeDB has support for tagged enums which can be found in languages like Ru
 
 To bridge the gap, a special marker interface `SpacetimeDB.TaggedEnum` can be used on any `SpacetimeDB.Type`-marked `struct` or `class` to mark it as a SpacetimeDB tagged enum. It accepts a tuple of 2 or more named items and will generate methods to check which variant is currently active, as well as accessors for each variant.
 
-It is expected that you will use the `Is*` methods to check which variant is active before accessing the corresponding field, as the accessor will throw an exception on a state mismatch.
+It is expected that you will use the `Is*` methods, or `.Tag` property to check which variant is active before accessing the corresponding field, as the accessor will throw an exception on a state mismatch.
```

Comment:
Oh no, `.Tag` was meant to be internal... either way, we have now made a larger breaking change to the tagged enum API, so this section will need to be reworked.